src/en/guide/get_started/setup.md

```diff
@@ -44,12 +44,115 @@ Remember: If you follow the `git clone` approach in [Installation](./installatio
 
 Here you are good to go! See [Quickstart](./quickstart) or our [Tutorials](/guide/tutorials/agent_101) for your first run!
 
-## Setup for different model providers
 
-### OpenAI
+## Detail Setting
```

Comment:
I recommend maintain the original subheader: Setup for different model providers.
The two sections do not differ in fast or detail but OpenAI API and other APIs